crates/sncast/src/starknet_commands/script.rs

```diff
@@ -300,15 +297,7 @@ pub fn run(
         .assemble_ex(&entry_code, &footer);
 
     // hint processor
-    let block_context = build_default_block_context();
-    let account_context = build_transaction_context();
-    let mut context = EntryPointExecutionContext::new(
-        &block_context.clone(),
-        &account_context,
-        ExecutionMode::Execute,
-        false,
-    )
-    .unwrap();
+    let mut context = build_context(BlockInfo::default());
```

Comment:
Did it bcs I moved `build_context` function to `runtime` package, so we don't repeat the code